rollbar/__init__.py

```diff
@@ -1007,6 +1014,34 @@ def _build_bottle_request_data(request):
 
     return request_data
 
+
+def _build_wsgi_request_data(request):
+    request_data = {
+        'url': wsgiref.util.request_uri(request),
+        'user_ip': _wsgi_extract_user_ip(request),
+        'method': request.get('REQUEST_METHOD'),
+    }
+    if 'QUERY_STRING' in request:
+        request_data['GET'] = urlparse.parse_qs(request['QUERY_STRING'], keep_blank_values=True)
```

Comment:
I just followed what all other request builders do. request_data['GET'] is used in _scrub_request_data.